CMSgov/design-system
# [WNMGDS-1596] responsive example

## Summary

* Made a `ResponsiveExample` component that shows a list of breakpoint buttons as well as an iframe to a storybook story with the example
  * Much of the responsive part of the iframe was taken from the original doc site implementation
  * Note, that iframed storybook stories will not show up when running `yarn start:gatsby`. This has something to do with gatsby building in production vs local dev mode
* Made a storybook story for responsive headings used on the typography page
* Created new yarn commands
  * `yarn build-storybook:gatsby` which builds storybook in production mode and puts it into the folder `packages/docs/static` which makes the files available by relative url to gatsby pages
  * `yarn serve:gatsby` serves the production build of the gatsby site

## How to test

To see the storybook items in action, run `yarn build-storybook:gatsby` to put storybook into gatsby's static folder. This only needs to be run once since storybook stories aren't changing.

To see gatsby typography page, run `yarn build:gatsby && yarn serve:gatsby`

![ResponsiveExample](https://user-images.githubusercontent.com/33579665/167219699-7daf2bad-141d-4d2b-9cb1-a9a24c473ada.gif)
